ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-release-4.18.yaml

```diff
@@ -50,32 +50,21 @@ releases:
       product: ocp
       stream: ci
       version: "4.18"
-  initial-417:
-    candidate:
-      product: ocp
-      relative: 1
-      stream: ci
-      version: "4.18"
   latest:
     integration:
       include_built_images: true
       name: 4.18-priv
       namespace: ocp-private
-  latest-417:
-    candidate:
-      product: ocp
-      stream: ci
-      version: "4.18"
   n1minor:
     candidate:
       product: ocp
       stream: ci
-      version: "4.18"
+      version: "4.17"
```

Comment:
Were these values just wrong before? Trying to understand why the value went down here and right below.

Reply:
Did a quick huddle with Patryk. These values were wrong before.